src/platform/forms-system/src/js/utilities/ui/index.js

```diff
@@ -10,19 +10,7 @@ export const $$ = (selector, root) => [
   ...(root || document).querySelectorAll(selector),
 ];
 
-export function focusElement(selectorOrElement, options) {
-  const el = $(selectorOrElement);
-
-  if (el) {
-    if (el.tabIndex === 0) {
-      el.setAttribute('tabindex', '0');
-    }
-    if (el.tabIndex < 0) {
-      el.setAttribute('tabindex', '-1');
-    }
-    el.focus(options);
-  }
-}
+export { focusElement };
```

Comment:
Trying to minimize duplicate code, but not need to remove this reference from all apps.